bin/src/main.rs

```diff
@@ -49,15 +52,33 @@ fn init_logging(cmd: &str) {
 
     let logfile = std::fs::OpenOptions::new()
         .create(true)
-        .truncate(true)
+        .truncate(false)
```

Comment:
Why set truncate to `true`?

Reply:
Probably to not make a big log file. But this make thing very inconvenient for debugging, because this code is run each time cosmic-launcher is toggled. Settings this to false didn't change anything for me somehow.
But i'm using journald now, but it doesn't support span unfortunately, which doesn't make it a perfect solution.

Reply:
Ahh sorry, I meant to say false. I think it should probably remain false, so that the log file doesn't keep growing.

Reply:
There is a condition some lines under:
```rs
if let Ok(meta) = file.metadata() {
            if meta.len() > 10000 {
                let _ = file.set_len(0);
            }
        }
```